hadoop-plugin/src/main/groovy/com/linkedin/gradle/hdfs/HdfsFileSystem.groovy

```diff
@@ -192,4 +192,30 @@ class HdfsFileSystem {
     throws IOException {
     fs.copyFromLocalFile(delSrc, overwrite, src, dst);
   }
+
+  /**
+   * @param p path to check
+   * @return true if p exists else returns false
+   */
+  public Boolean exists(Path p) {
+    fs.exists(p);
```

Comment:
Can you add the "return" keyword to these methods, even though it's not required in Groovy?
Please make these return little boolean instead of Boolean